### PR TITLE
Update bot app creation flow

### PR DESCRIPTION
**Pull request by queezly**
Discord changed the Developer section of their website
This PR updates the relevant guide section to reflect these changes and adds an image and danger-box on "how to reset token in case of leak"

### CODE REVIEW

File: guide/preparations/setting-up-a-bot-application.md [hunk 1]
```diff
@@ -5,25 +5,33 @@ Now that you've installed Node, discord.js, and hopefully a linter, you're almos
 It's incredibly easy to create one. The steps you need to take are as follows:
 
 1. Open up [the Discord website](https://discordapp.com/) and [login](https://discordapp.com/login).
-2. Hover over the "More" drop-down menu and click on the [Developers](https://discordapp.com/developers/docs/intro) link.
-3. On the sidebar, under "Applications", click on the [My Apps](https://discordapp.com/developers/applications/me) link.
-4. Click on the "New App" button and fill out the form. **Note:** Only the app name field is required; all the others are optional. Submit the form once you're done.
+2. Hover over the "Developers" drop-down menu and click on the [Developer Portal](https://discordapp.com/developers/docs/intro) link.
+3. On the header click on the [Applications](https://discordapp.com/developers/applications) link.
+4. Click on the "Create an application" button.
```
Comment (PLASMAchicken) [inline guide/preparations/setting-up-a-bot-application.md]: Isn't here a unnecessary space

Comment (queezly) [reply guide/preparations/setting-up-a-bot-application.md]: fixed

File: guide/preparations/setting-up-a-bot-application.md [hunk 2]
```diff
@@ -39,4 +47,10 @@ Let's imagine that you have a bot on over 1,000 servers, and it took you many, m
...
 * Make your bot leave all of the servers it has joined.
 
-All that and much, much more. Sounds pretty terrible, right? So make sure to keep your token as safe as possible! Now, let's move on to the next step.
+All that and much, much more. Sounds pretty terrible, right? So make sure to keep your token as safe as possible!
+
+<p class="danger">Should you ever leak your token, commit it to a repository, post it in a support channel, see your brother take a picture of it or otherwise see your bot in danger return to this page and click the "Regenerate" button.</p>
+
+![Reset Token](assets/img/token-reset.png)
+
+After you regenerated your token the old one will be invalid and you will ned to update it whereever you use it!
```
Comment (PLASMAchicken) [inline guide/preparations/setting-up-a-bot-application.md]: >After you regenerated your token the old one will be invalid and you will ned to update it **~~whereever~~** you use it!

https://www.dict.cc/?s=wherever

Comment (PLASMAchicken) [inline guide/preparations/setting-up-a-bot-application.md]: >After you regenerated your token the old one will be invalid and you will **~~ned~~** to update it whereever you use it!

Comment (queezly) [reply guide/preparations/setting-up-a-bot-application.md]: fixed both

File: guide/preparations/setting-up-a-bot-application.md [hunk 3]
```diff
@@ -29,9 +28,9 @@ After creating a bot user, you'll see a section like this:
 
 ![Bot application](assets/img/created-bot.png)
 
-In this panel you can set a snazzy picture, set a username for your bot and make it public or private.
-There is also a section labeled "Token"
-Click the "copy" button to copy your token. When we ask you to paste your token somewhere, this is the value that you need to put in. Don't worry if you do happen to lose your token. You can always go back to this page and copy it again, as well as generate a new one, if necessary.
+In this panel you can add a snazzy profile picture, set a username for your bot and make it public or private.
+You can access your token in this panel as well either by reavealing it or simply pressing the "copy" button.
```
Comment (PLASMAchicken) [inline guide/preparations/setting-up-a-bot-application.md]: >You can access your token in this panel as well either by reavealing it or simply pressing the "copy" button.

revealing

**Pull request merged**